tests/unit/test_knob_extractor.py

```diff
@@ -437,6 +443,24 @@ def test_extractor_in_cern_network(self, tmp_path, knob_definitions, saved_knobf
         for key in parsed_output.keys():
             assert parsed_output[key] == parsed_saved[key]
 
+    @pytest.mark.cern_network
+    def test_state_in_cern_network(self, state_tfs):
+        # get recorded data
+        old_df = tfs.read_tfs(state_tfs, index="NAME")
+        time = old_df.headers[Head.time]
+
+        # run main
+        state_df = main(time=time, state=True)
+
+        # format a bit to make frames equal
+        state_df = state_df.applymap(str)
+        state_df.headers[Head.time] = str(state_df.headers[Head.time])
+        state_df.index.name = "NAME"
+
+        # check frames
+        assert_dict_equal(state_df.headers, old_df.headers)
+        assert_frame_equal(state_df, old_df)  # dframe_y already pandas
```

Comment:
That's not really gonna work everytime, is it? What if we're not on 30cm optics? Best we can do I guess?

Reply:
It is. it checks it at a given time-stamp, which is always the same.